app/src/main/java/com/keylesspalace/tusky/components/account/media/AccountMediaFragment.kt

```diff
@@ -133,6 +148,27 @@ class AccountMediaFragment :
         }
     }
 
+    override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
```

Comment:
AccountMediaFragment and other fragments in the AccountActivity should only be refreshed together. If we are doing this we need to at least be consistent

MainActivity: Every Fragment has an individual swipe to refresh. The new menu refreshes the individual, active fragment ✅ 
AccountActivity: There is one swipe to refresh. It refreshes the activity as well as all the fragments. The menu options just refreshes the active fragment now 🟥 If it is supposed to improve the accessibilty of the swipe to refresh, it needs to refresh the same thing.

Actually when I think about it, having the menu option refresh all fragments, even in MainActivity and SearchActivity, could be a good idea and would lessen my concerns of it being confusing.

Reply:
> AccountActivity: There is one swipe to refresh. It refreshes the activity as well as all the fragments. The menu options just refreshes the active fragment now 🟥 If it is supposed to improve the accessibilty of the swipe to refresh, it needs to refresh the same thing.

That makes sense. It's an oversight that it doesn't. I'll fix that.

> Actually when I think about it, having the menu option refresh all fragments, even in MainActivity and SearchActivity, could be a good idea and would lessen my concerns of it being confusing.

That contradicts the previous paragraph, where you just said "it needs to refresh the same thing2.

The menu should behave *identically* to the swipe gesture.